trading_calendars/exchange_calendar_xphs.py

```diff
@@ -59,6 +59,13 @@ def only_friday(dt):
 
 ChineseNewYearAfter2011 = ChineseNewYear[ChineseNewYear.year > 2011]
 
+PeoplePowerRevolution = Holiday(
+    'People Power Revolution',
+    month=2,
+    day=25,
+    start_date=2016,
```

Comment:
Sorry for missing this before, but in the test csv for XPHS it looks like February 25th got treated as a holiday all the way back to 1990. My guess is that `2016` needs to be a string:

```suggestion
    start_date='2016',
```

Reply:
yikes, will fix, thanks